### PR TITLE
Fix depreciations in tests

### PR DESCRIPTION
**Pull request by adamslc**
Just a few quick and easy fixes to make the tests stop complaining.

### CODE REVIEW

**Comment by timholy** [comment]
Another option is to adopt 0.7 wholesale. I'd be fine with either approach.

**Comment by martinholters** [comment]
Dropping 0.6 support completely would also be fine with me. The changes here to keep compatibility with 0.6 appear to be so small, however, that it might be _less_ work than getting rid of Compat entirely. I leave it up to you, @adamslc.

**Comment by adamslc** [comment]
OK, I've gone ahead and fully transitioned to 0.7.

**Comment by timholy** [comment]
Thanks! Can you also remove the 0.6 line from `.travis.yml`? Then I think we can merge this.

File: REQUIRE [hunk 1]
```diff
@@ -1,2 +1 @@
-julia 0.6
-Compat 0.55.0
+julia 0.7-
```
Comment (martinholters) [inline REQUIRE]: Not that it matters much, but should we rather set this to `0.7.0-alpha`?

**Comment by timholy** [comment]
Thanks, @adamslc (and @martinholters for his input here).

File: REQUIRE (hunk 1 above)
Comment (timholy) [reply REQUIRE]: Hmm, now that I think about it I guess this allows early dev versions too. Well, I don't really think anyone would be using anything older than the alpha anyway...

**Pull request merged**